### ISSUE TITLE
SpecialFunctions with Multiple Versions of Julia

### ISSUE THREAD

**Issue by matthieugomez** [JuliaMath/SpecialFunctions.jl#174]
I have both Julia 1.1 and Julia 1.2 installed on my computer (MacOS). Everytime I switch between the two, I need to rebuild SpecialFunctions. Would there be a way to avoid this?
```julia
ERROR: LoadError: InitError: ~/.julia/packages/SpecialFunctions/fvheQ/deps/usr/lib/libopenspecfun.1.3.dylib cannot be opened, Please re-run Pkg.build("SpecialFunctions"), and restart Julia.
```

**Comment by ararslan** [comment]
I can't reproduce on macOS or Linux; switching between versions of Julia recompiles the package upon `using SpecialFunctions` but works just fine.

**Comment by iamed2** [comment]
Happens to @rofinn and others often

**Comment by rofinn** [comment]
I think it depends on which versions of julia you're switching between. I believe I've noticed this switching between 1.0 and 1.2/1.3.

**Comment by musm** [comment]
Doesn't really seem an issue for `SpecialFunctions` specifically, but rather Julia or Pkg related

**Comment by iamed2** [comment]
Rather, I think this was fixed by migrating to Julia 1.3 and OpenSpecFun_jll.

**Comment by mcfefa** [comment]
I just ran into this when upgrading from Julia 1.0 to Julia 1.4.

```julia
julia>  using SpecialFunctions
ERROR: InitError: /Users/.../.julia/packages/SpecialFunctions/ne2iw/deps/usr/lib/libopenspecfun.1.3.dylib cannot be opened,
Please re-run Pkg.build("SpecialFunctions"), and restart Julia.
```

Running `Pkg.build("SpecialFunctions")` fixed it.
